qutebrowser/browser/webkit/webview.py

```diff
@@ -72,6 +72,12 @@ def __init__(self, win_id, tab_id, tab, parent=None):
 
         page = webpage.BrowserPage(self.win_id, self._tab_id, tab.data,
                                    parent=self)
+
+        if qtutils.version_check('5.2'):
+            page.setVisibilityState(
+                page.VisibilityStateVisible if self.isVisible()
```

Comment:
While this works, I'd prefer accessing enum members like `QWebPage.VisibilityStateVisible` because they aren't really things defined on the `page` instance.